README.md

```diff
@@ -674,6 +674,26 @@ bat() {
 
 ## Troubleshooting
 
+### Garbled output
+
+As `bat` processes output its own way in order to provide syntax highlighting
+and print decorations, most output that is composed by escape sequences just
+won't work. That is output:
+
+1. That's already coloured (eg. compiler warnings output).
+
+   `bat` [defaults to using `less
+   -RFX`](https://github.com/sharkdp/bat#using-a-different-pager), but it also
+   uses its own decorations, so it's not strictly compliant with output that
+   would work on `less -R` alone.
+
+2. That needs reprinting (eg. a progress bar).
+
+   `bat` can only work as a drop-in `cat` replacement whenever it detects a
+   non-interactive terminal session, so you might be able to make it work by
+   using `bat -pp`, if that doesn't suit your needs, please check out the man
+   pages for more options.
+
```

Comment:
In light of what we talked about recently, what do you think about shortening this section and just having it say the following?

```suggestion
### Garbled output

If an input file contains color codes or other ANSI escape sequences, `bat` will
have troubles to perform syntax highlighting and text wrapping. The output can
become garbled. When displaying such files it is recommended to disable both
syntax highlighting and wrapping by passing `--color=never --wrap=never` to
`bat`.

```

Reply:
Sure, that's a lot cleaner and does seem to work :)

I'll only change a couple words for clarity.